### DISCUSSION TITLE
show the invite to server button on bot profile

### DISCUSSION THREAD

**Discussion by Error404Hacked** [SudhanPlayz/Discord-MusicBot#596] [Q&A] [answered]
op original question: How to show the invite to server button on discord profile

**Comment by Shadoukita** [comment]
That has nothing to do with the Bot code.

That's something you set up on the discord applications.

**Comment by DarrenOfficial** [comment, marked as answer]
It’s in the oauth page of the discord developer portal
![oauth page correct](https://user-images.githubusercontent.com/55406094/145129920-8ab96b1a-c50c-46b7-b13c-3d143e92837c.jpeg)
